### PR TITLE
feat: allow to copy evaluate command in a CLI format

### PR DESCRIPTION
**Pull request by kyryl-perepelytsia**
Related to https://github.com/flipt-io/flipt/issues/2863

### CODE REVIEW

**Comment by kyryl-perepelytsia** [comment]
hey @GeorgeMac, sorry my bad, i have re-pushed all changes

**Comment by markphelps** [comment]
@all-contributors please add @kyryl-perepelytsia for code

**Comment by markphelps** [comment]
Works great @kyryl-perepelytsia ! Thank you for the PR! one minor bit of feedback, I think the spacing between the buttons is a bit off in the UI

In 'desktop' view I feel like the spacing is too large between the button and the dropdown

Xnapper-2024-06-10-09 38 26

In mobile view there is some overlap. 

Screen Shot 2024-06-10 at 09 36 35

wdyt?

File: ui/src/app/console/Console.tsx [hunk 1]
```diff
@@ -133,6 +137,34 @@ export default function Console() {
...
+        { key: '--entity-id', value: values.entityId },
+        { key: '--namespace', value: namespace.key },
+        {
+          key: '--context',
+          value: Object.entries(parsed)
+            .map(([key, value]) => `${key}=${value}`)
+            .join(' ')
+        }
+      ]
+    });
```
Comment (erka) [inline ui/src/app/console/Console.tsx:163]: Thank you for your initial contribution @kyryl-perepelytsia.

There is a small adjustment needed for the CLI command regarding the context option. If there are multiple context values, each one should be passed as a separate option, such as `--context apple=a --context pear=p`.

```suggestion
    const contextOptions = Object.entries(parsed).map(([key, value]) => {
      return {
        key: '--context',
        value: `${key}=${value}`
      };
    });
    const command = generateCliCommand({
      commandName: Command.Evaluate,
      arguments: [values.flagKey],
      options: [
        { key: '--entity-id', value: values.entityId },
        { key: '--namespace', value: namespace.key },
        ...contextOptions
      ]
    });
```

Comment (kyryl-perepelytsia) [reply ui/src/app/console/Console.tsx]: Thanks, updated

**Review by markphelps — APPROVED** [review]
looks great! thank you @kyryl-perepelytsia . I'll see if I can adjust the button gaps in a follow up

**Comment by kyryl-perepelytsia** [comment]
> looks great! thank you @kyryl-perepelytsia . I'll see if I can adjust the button gaps in a follow up

oh you already merged it, thanks! 

Actually i am bit struggled with a positioning of this button. 

The problem in dropdown that has absolute positioning, and with a sticky footer. 

If i'm trying to apply relative position on some of a parent elements inside form (to make correct positioning of a button with dropdown), it will correctly place button, but there are other issues with dropdown(depends on what parent you adding relative position) : 
1) It starting to require scrolling down, as dropdown do not overlap footer (and its super non intuitive that button is in open state, as its hided under footer, and shows only after you scroll the parent DIV of a button)
2) The dropdown also do not overlap footer, but the whole form starting to jump when you opening drop down

**Pull request merged**